src/app/components/account/account-avatar/account-avatar.tsx

```diff
@@ -8,18 +8,38 @@ interface AccountAvatarProps extends BoxProps {
   name: string;
 }
 export const AccountAvatar = memo(({ publicKey, name, ...props }: AccountAvatarProps) => {
+  let firstGrapheme: string = '';
+  if (name?.includes('Account')) {
+    // Displays the account number.
+    //
+    // Assumes the name of the account is similar to "Account 4". This will
+    // probably break when the account has a BNS name with the string
+    // `"Account"` in it, although it's probably good enough for now.
+    firstGrapheme = name.split(' ')[1];
+  } else {
+    // Displays the first grapheme of the account name in uppercase.
+    //
+    // Using graphemes allows proper handling of emojis and other characters
+    // with multiple code points.
+    firstGrapheme = [...new Intl.Segmenter().segment(name)][0].segment;
+
+    // Will only transform text characters. E.g., emojis are unaffected.
+    firstGrapheme = firstGrapheme.toUpperCase();
+  }
+  const isFirstGraphemeEmoji = /\p{Extended_Pictographic}/u.test(firstGrapheme);
+
   const gradient = useAccountGradient(publicKey ?? '');
 
-  const circleText = name?.includes('Account') ? name.split(' ')[1] : name[0];
   return (
     <Circle
       flexShrink={0}
-      backgroundImage={gradient}
-      color={color('bg')}
+      background={isFirstGraphemeEmoji ? '#f4f4f6' : color('bg')}
+      backgroundImage={isFirstGraphemeEmoji ? undefined : gradient}
+      color={color('text-title')}
       userSelect="none"
       {...props}
     >
-      {circleText.toUpperCase()}
+      {firstGrapheme}
```

Comment:
Is grapheme the right word to use? 🤔  Are we referring to a glyph/character?

Reply:
I think it is, I took it from the `Segmenter` [options](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Segmenter/Segmenter#parameters) (which defaults to segmenting strings by `grapheme`). Using glyph/character may be more understandable, although they don't include all possible values the grapheme may take (eg, emojis).